GUI/src/cz/fidentis/gui/comparison_batch/BatchRegistrationConfiguration.java

```diff
@@ -1391,6 +1419,17 @@ public void setConfiguration() {
        /* if(c.getModels().size() > 0)
             facesComboBox.setSelectedIndex(c.getTemplateIndex() + 3);
        */
+       
+       //to check whether FPs can be exported once they are added, removed
+        ObservableMaster o = new ObservableMaster();
+        ExportFPButtonObserver export = new ExportFPButtonObserver(exportFpButton, 
+        GUIController.getSelectedProjectTopComponent().getViewerPanel_Batch().getListener(),
+        null);
+        RegisterFPButtonObserver register = new RegisterFPButtonObserver(registerButton, c.getFacialPoints());
+        o.addObserver(export);
+        o.addObserver(register);
+        
+        GUIController.getSelectedProjectTopComponent().getViewerPanel_Batch().setFpExportEnable(o);
```

Comment:
Nedalo by sa pri kazdom zavolani tejto funkcie len nejak zmenit kontext toho observable aparatu a nevytvarat to vsetko nanovo? Obavam sa ci vytvaranie vzdy novych instancii tychto veci nesposobi to ze budu zostavat niekde visiet a ak bude mat clovek dlho otvoreny program moze to sposobit problemy.